src/Raven.Studio/typescript/widgets/virtualGrid/columns/nodeTagColumn.ts

```diff
@@ -1,21 +1,33 @@
 ﻿/// <reference path="../../../../typings/tsd.d.ts"/>
 
 import virtualGridController = require("widgets/virtualGrid/virtualGridController");
-import hyperlinkColumn = require("./hyperlinkColumn");
+import hyperlinkColumn = require("widgets/virtualGrid/columns/hyperlinkColumn");
 
 class nodeTagColumn<T extends { nodeTag: string, database: string, noData: boolean }> extends hyperlinkColumn<T> {
+
+    hrefProvider: (item: T) => { url: string; openInNewTab: boolean; targetDescription?: string };
     
-    constructor(gridController: virtualGridController<any>, hrefProvider: (item: T) => { url: string; openInNewTab: boolean }) {
+    constructor(gridController: virtualGridController<any>, 
+                hrefProvider: (item: T) => { url: string; openInNewTab: boolean; noData: boolean; targetDescription?: string }) {
         super(gridController, item => this.valueProvider(item), item => hrefProvider(item).url, "Node", "70px", {
             useRawValue: () => true,
             openInNewTab: item => hrefProvider(item).openInNewTab
         });
+        
+        this.hrefProvider = hrefProvider;
     }
     
     private valueProvider(item: T) {
         const nodeTag = item.nodeTag;
         const extraClass = item.noData ? "no-data" : `node-${nodeTag}`;
-        return `<span class="node-label ${extraClass}">${nodeTag}</span>`;
+        
+        if (item.nodeTag) {
+            const description = this.hrefProvider(item).targetDescription;
+            const titleText = description ? `Go to ${description} view` : "";
```

Comment:
not sure assuming that description can only navigate between views is good. I'd rather keep it generic and put description user gave w/o any extras.

Reply:
done - removed the 'view' part from here